pkg/tokenutil/create_secret_key.go

```diff
@@ -0,0 +1,142 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package tokenutil
```

Comment:
any idea why this package is not under `pkg/ctl`?

Reply:
In the Pulsar project. The token util is an independent command and it is not included in the pulsar-admin. In the package `pkg/ctl`, all commands are used to operate the Pulsar cluster or namespace or topic and so on.  The token tools just used to create some token related files. So maybe make it as an independent command is better?

Reply:
for providing a smooth experience for the users, it is making more sense to add token as a sub command of `pulsarctl`. so people can use `pulsarctl token` to generate tokens. if you take a look at `kubectl`. `kubectl` also has commands that do local operations which don't talk to a cluster.